packages/core/src/components/text-input/text-input.css

```diff
@@ -258,18 +258,21 @@
   display: flex;
   align-items: center;
   justify-content: center;
-  margin-right: var(--spacing-s);
+  margin-right: calc(var(--spacing-s) - var(--spacing-xs) / 2);
   font-size: 1rem;
 }
 
 .hds-text-input__button {
-  padding: 0;
   appearance: none;
-  background: transparent;
+  background: none;
+  border: none;
   cursor: pointer;
-  border: 0;
-  width: 1.5rem;
-  height: 1.5rem;
-  overflow: hidden;
-  font-size: 1rem;
+  font: inherit;
+  outline: none;
+  display: flex;
+  padding: var(--spacing-xs) calc(var(--spacing-xs) / 2);
```

Comment:
The button area would be bigger if we could use just:
`padding: var(--spacing-s);` and remove the `margin-right`
The current implementation:

<img width="555" alt="Screenshot 2022-11-22 at 9 24 05" src="https://user-images.githubusercontent.com/1610860/203251471-1fbbcfd8-2b2a-416d-b86a-07f197e07a78.png">

With the changes:
<img width="490" alt="Screenshot 2022-11-22 at 9 23 43" src="https://user-images.githubusercontent.com/1610860/203251541-d29528d0-5bdf-4524-a862-e09732437f21.png">

**But** this means that the layout of the Search Input changes a bit because there are two buttons when the input has a value. They require more space now.
So it is either or a **breaking change** or we just accept that the change is so small that it does not require major release. Or we leave this as it is.

Reply:
<img width="520" alt="Screenshot 2022-11-22 at 9 33 44" src="https://user-images.githubusercontent.com/1610860/203252919-8e99f76c-26c8-4448-b466-d3014cbf1118.png">
<img width="530" alt="Screenshot 2022-11-22 at 9 32 38" src="https://user-images.githubusercontent.com/1610860/203252922-46feb690-7e70-429f-ba5f-642fff87f24c.png">